src/components/auth/Input.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { TextInput, View, Text, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    height: 40,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    paddingVertical: 25,
+    marginHorizontal: 15,
+    borderBottomWidth: 2,
+    borderColor: 'white',
+  },
+  labelStyle: {
+    flex: 1,
+    fontSize: 14,
+    fontWeight: 'bold',
+    color: 'white',
+    backgroundColor: 'transparent',
+  },
+  inputStyle: {
+    flex: 2,
+    height: 40,
+    fontSize: 14,
+    fontWeight: 'bold',
+    color: 'white',
+    paddingHorizontal: 5,
+  },
+});
+
+const Input = ({
+  label,
+  value,
+  onChangeText,
+  placeholder,
+  secureTextEntry,
+  keyboardType,
+}) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.labelStyle}>{label}</Text>
+      <TextInput
+        keyboardType={keyboardType}
+        secureTextEntry={secureTextEntry}
+        placeholder={placeholder}
+        placeholderTextColor="white"
```

Comment:
can you make this color `rgba(255,255,255,0.5)`

Reply:
ok